includes/class-sensei-utils.php

```diff
@@ -365,6 +375,11 @@ public static function sensei_get_activity_value( $args = array() ) {
 	 * @return bool
 	 */
 	public static function sensei_customer_bought_product( $customer_email, $user_id, $product_id ) {
+		_deprecated_function( __METHOD__, '1.9.0', 'Sensei_WC::has_customer_bought_product($user_id, $product_id)' );
```

Comment:
Looks like we're still calling this function [here](https://github.com/Automattic/sensei/blob/15e446478abe2b4841d1b4390837d6868a6b0340/includes/class-sensei-course.php#L2385).

Reply:
That's to be removed in https://github.com/Automattic/sensei/pull/2406

Reply:
It feels like a misnomer to use 1.9.0 in that case as it's technically being deprecated in 2.0.0.

Reply:
It was marked as deprecated by docblock notation in 1.9.0, we just weren't warning people until now.